ddl/db_test.go

```diff
@@ -483,31 +483,8 @@ func testCancelDropIndex(c *C, store kv.Storage, d ddl.DDL, idxName, addIdxSQL,
 	hook.OnJobRunBeforeExported = func(job *model.Job) {
```

Comment:
The point here is to resolve code conflict. Could you elaborate under https://github.com/pingcap/tidb/issues/28636 about which test you'd like to rewrite so that I can migrate other tests first and avoid code conflict?